aliases/available/vagrant.aliases.bash

```diff
@@ -2,13 +2,16 @@ cite 'about-alias'
 about-alias 'vagrant aliases'
 
 # Aliases
+alias vhl='vagrant hosts list'
+alias vscp='vaagrant scp'
+alias vsl='vagrant snapshot list'
+alias vst='vagrant snapshot take'
 alias vup="vagrant up"
 alias vupl="vagrant up 2>&1 | tee vagrant.log"
 alias vh="vagrant halt"
 alias vs="vagrant suspend"
 alias vr="vagrant resume"
 alias vrl="vagrant reload"
-alias vssh="vagrant ssh"
```

Comment:
I made another error here in unintentionally removing this. I already have a function in my  .bashrc so that I don't have to su:

function vssh() {
    vagrant ssh $1 -- -t 'sudo su -; /bin/bash'
}